src/components/app.cjsx

```diff
@@ -97,18 +116,22 @@ module.exports = React.createClass
         <div className="container-fluid">
           <div className="navbar-header">
             <BS.ButtonToolbar className="navbar-btn">
-              <a href="/exercises/new" className="btn btn-success">New Blank Exercise</a>
-              <AsyncButton
-                bsStyle='info'
-                onClick={@saveExercise}
-                disabled={isWorking}
-                isWaiting={ExerciseStore.isSaving(id)}
-                waitingText='Saving...'
-                isFailed={ExerciseStore.isFailed(id)}
-                >
-                Save Draft
-              </AsyncButton>
-              { if not ExerciseStore.isNew(id)
+              <a href="/exercises" onClick={@onReset} className="btn btn-danger">Reset</a>
```

Comment:
I'm sorry!  I meant to reply to this yesterday and had not.  Yes, it is selectable by tab+space (it's an A tag). 

Prompt before unsaved changes is there, but is going to have to be fine-tuned with buttons being disabled when appropriate since there's an new bug-report on it.